### ISSUE TITLE
[BUG] Exporting To Dos as CSV generates a non-usable file with HTML content

### ISSUE THREAD

**Issue by piogiuse** [Leantime/leantime#2042]
*If you have problems with the installation please use our [community forum](https://community.leantime.io)*

**What set up are you using**
* [ ] Cloud Hosted Version
* [x] Self Hosted

**Describe the bug**
When exporting To Dos as csv, the downloaded file contains HTML code and is not usable.

**To Reproduce**
1. Open a project
2. Click on 'To Dos'
3. Select the table-view
4. Click on 'export'
5. Open the downloaded csv-file

**Expected behavior**
A csv file being created containing all To Dos.

**Leantime Version**
2.4.4

**Server**
Apache

**PHP / MySQL Version**
PHP 8.1 / 10.5.22-MariaDB

**Additional context**
none

**Comment by gloriafolaron** [comment]
Thanks for reporting this! 

This should actually not yet be visible so we appreciate the call out here and we'll work to correct that until there is a more functional solution. 

Could you share with me what the goal or what you are hoping to accomplish with the csv export? 

I ask as we want to make sure that, however we address the solution, it helps to accomplish what you're actually trying to do.

**Comment by piogiuse** [comment]
Thanks for letting me know!

Of course. The reason I'd like to export my to-dos as CSV is to create a project plan formatted the way I need it.
Currently, I manually take screenshots of the timeline as well as of the to-dos and put them into a powerpoint presentation.

It would be great to export a single PDF including the timeline, to-dos, and an overview of the progress on milestones (currently available under 'data room' -> 'reports').

**Comment by gloriafolaron** [comment]
Thanks for highlighting that!

Would you remind me what setting you use Leantime in? As in, are the screenshots stakeholder updates, client updates, etc?

And are the folks you're doing the SS for - are they in the system?

Our AI plugin will have a status update email to go out but I recently had some conversations that had me looking at a more visual timeline and task based "stakeholder/client" report. It'd be more get approvals/cleared focused than update on the week's work focused. I'm curious if they have some overlap.

**Comment by piogiuse** [comment]
Of course! I use Leantime for managing university projects. As for stakeholders, there are students as well as professors. The screenshots are used to give an overview of the current project status (e.g., during mid-term presentations, status updates, etc.) as well as for final project reports.

They are not in Leantime, but we are currently thinking about creating a read-only account for them to check the progress / current project status autonomously.

That sounds great! I would love to have a mix of a text-based update as well as a graphical update, as in my opinion graphics are more powerful for product owners / when it comes to providing quick but detailed overviews.
